components/patient-access-rules/ui/src/main/resources/PhenoTips/AccessRightsManagement.xml

```diff
@@ -435,8 +435,9 @@
         "collaborator"  : "$escapetool.javascript($services.localization.render('phenotips.accessRightsManagement.collaboratorRole'))",
         "user"          : "$escapetool.javascript($services.localization.render('phenotips.accessRightsManagement.userType'))",
         "group"         : "$escapetool.javascript($services.localization.render('phenotips.accessRightsManagement.groupType'))",
-        "private"       : "$escapetool.javascript($services.localization.render('phenotips.permissions.visibility.private.label'))",
-        "hidden"       : "$escapetool.javascript($services.localization.render('phenotips.permissions.visibility.hidden.label'))",
+#foreach ($visibility in $services.permissions.listVisibilityOptions())
```

Comment:
Actually, there may be records that still use one of the disabled visibilities, so it would be safer to list all the visibility options:

```suggestion
#foreach ($visibility in $services.permissions.listAllVisibilityOptions())
```